### PR TITLE
feat(rum): add advanced RUM sampling

### PR DESCRIPTION
**Pull request by trieloff**
includes tracking of `viewmedia` and `viewblock`

https://advanced-rum--express-website--adobe.hlx3.page/express/?rum=on&lighthouse=on

### CODE REVIEW

**Review by rofe — APPROVED** [review]

**Comment by rofe** [comment]
@trieloff feel free to merge this after resolving the conflict

**Comment by trieloff** [comment]
Merged and updated. I don't know why the `delayed.js` disappeared. As @davidnuescheler is still holding out, let's wait for his approval, too.

**Comment by rofe** [comment]
> Merged and updated. I don't know why the `delayed.js` disappeared. 

Looks like it became `gnav.js` in [this commit](https://github.com/adobe/express-website/commit/1c1280e30eebe6172a4828aef1f3a154ee7c8712)

**Review by davidnuescheler — APPROVED** [review]

**Pull request merged**
